samples/snippets/storage_transfer_manager_upload_directory.py

```diff
@@ -30,12 +30,13 @@ def upload_directory_with_transfer_manager(bucket_name, source_directory, thread
     # working directory".
     # source_directory=""
 
-    # The number of threads to use for the operation. The performance impact of
-    # this value depends on the use case, but generally, smaller files benefit
-    # from more threads and larger files don't benefit from more threads. Too
-    # many threads can slow operations, especially with large files, due to
-    # contention over the Python GIL.
-    # threads=4
+    # The maximum number of worker processes that should be used to handle the
```

Comment:
Since this sample doesn't mention the worker type constants, let's try omitting them.

"The maximum number of processes to use for the operation. The performance impact of this value depends on the use case, but smaller files usually benefit from a higher number of processes. Each additional process occupies some CPU and memory resources until finished."

Reply:
Omitted worker type in comments related to processes. PTAL.